ActiveForm.php

```diff
@@ -98,4 +97,13 @@ public function init()
         }
         parent::init();
     }
+
+    /**
+     * @inheritdoc
+     * @return ActiveField the created ActiveField object
+     */
+    public function field($model, $attribute, $options = [])
```

Comment:
Isn't it safe to remove it? It does nothing other than calling parent implementation.